docs/framework/migration-guide/net-compatibility-diagnostics.md

```diff
@@ -460,7 +460,7 @@ The .NET Compatibility Diagnostics are Roslyn-powered analyzers that help identi
 |Scope|Edge|  
 |Version|4.5|  
 |Type|Runtime|  
-|Affected APIs|<xref:System.Linq.Enumerable.Empty%60%601?displayProperty=fullName>|  
+|Affected APIs|<xref:System.Linq.Enumerable.Empty%60%601%2A?displayProperty=fullName>|  
```

Comment:
Yep ... let me revert. The build report lied ... the original link works. https://docs.microsoft.com/en-us/dotnet/articles/framework/migration-guide/net-compatibility-diagnostics#42-enumerableemptytresult-always-returns-cached-instance